agents/threadpool/hippo4j-agent-bootstrap/src/main/java/cn/hippo4j/agent/bootstrap/Hippo4jAgent.java

```diff
@@ -49,18 +48,17 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.Map;
-import java.util.concurrent.ArrayBlockingQueue;
-import java.util.concurrent.ExecutorService;
-import java.util.concurrent.ThreadPoolExecutor;
-import java.util.concurrent.TimeUnit;
 
 import static net.bytebuddy.matcher.ElementMatchers.nameContains;
 import static net.bytebuddy.matcher.ElementMatchers.nameStartsWith;
 import static net.bytebuddy.matcher.ElementMatchers.not;
 
+/**
+ * hippo for java Agent
+ */
 public class Hippo4jAgent {
 
-    private static ILog LOGGER = LogManager.getLogger(Hippo4jAgent.class);
+    private static ILog logger = LogManager.getLogger(Hippo4jAgent.class);
```

Comment:
Can it be capitalized if it is defined as final?

Reply:
no，beacuse the mehtod 'Permian' has assigned value for it.